fiboa_cli/datasets/fs_br.py

```diff
@@ -0,0 +1,206 @@
+from ..convert_utils import convert as convert_
+import re
+import pandas as pd
+
+# Please REMOVE the comment after review: 
+# I was actually using this link "https://prod-dcd-datasets-cache-zipfiles.s3.eu-west-1.amazonaws.com/vz6d7tw87f-1.zip" which is copied from the 
+# "Download All" button from the website (https://data.mendeley.com/datasets/vz6d7tw87f/1). 
+# But output an error saying """/vsizip//tmp/tmp7puy6g90/vz6d7tw87f-1.zip' not recognized as a supported file format. 
+# It might help to specify the correct driver explicitly by prefixing the file path with '<DRIVER>:', e.g. 'CSV:path'""""
+SOURCES = "https://github.com/fiboa/data/files/15438544/LEM_dataset.zip"
+
+
+ID = "br"
+SHORT_NAME = "Brazil"
+TITLE = "Field boundaries for Brazil"
+DESCRIPTION = """
+This dataset is the supplementary data of a paper published in the Data in Brief Journal. 
+
+The dataset, in ESRI shapefile format (spatial reference system: WGS 84, EPSG: 4326), provides monthly land use 
+information about 1854 fields from October 2019 to September 2020 from Luís Eduardo Magalhães (LEM) and other 
+municipalities in the west of Bahia state, Brazil. The majority of the 16 land uses classes are related to crops.
+"""
+
+
+PROVIDERS = [
+    {
+        "name": "Mendeley Data",
+        "url": "https://data.mendeley.com/datasets/vz6d7tw87f/1#file-5ac1542b-12ef-4dce-8258-113b5c5d87c9",
+        "roles": ["producer", "licensor"]
+    }
+]
+
+ATTRIBUTION = "Copyright © 2024 Elsevier inc, its licensors, and contributors."
+LICENSE = "CC-BY-4.0"
+
+# Please REMOVE the comment after review: 
+# The commented variables are my proposed columns attributes, but it doesn't seem 
+# logical to me to have months and years as the data's attributes. What do you think?
+COLUMNS = {
+    'geometry': 'geometry',
+    'id': 'id',
+    # 'OCT_2019': 'oct_2019',
+    # 'NOV_2019': 'nov_2019',
+    # 'DEC_2019': 'dec_2019',
+    # 'JAN_2020': 'jan_2020',
+    # 'FEB_2020': 'feb_2020',
+    # 'MAR_2020': 'mar_2020',
+    # 'APR_2020': 'apr_2020',
+    # 'MAY_2020': 'may_2020',
+    # 'JUN_2020': 'jun_2020',
+    # 'JUL_2020': 'jul_2020',
+    # 'AUG_2020': 'aug_2020',
+    # 'SEP_2020': 'sep_2020',
+    # 'NOTE': 'note'
+}
+
+# Please REMOVE the comment after review: 
+# I do think it is safe to assume the determination date is the last date of September in 2020 as the dataset
+# has SEP_2020 (which is the latests time) as one of its column attributes.
+ADD_COLUMNS = {
+    "determination_datetime": "2020-09-30T00:00:00Z"
+}
```

Comment:
```suggestion
ADD_COLUMNS = {}
```

If we are uncertain about the determination datetime we can also simply not provide it, it's not required.
I feel like it's better to have no data rather than potentially incorrect data.
I guess we'd need to read the paper to get certainty around this.